### ISSUE TITLE
Allow schema on resources, not just packages

### ISSUE THREAD

**Issue by rossjones** [datagovuk/ckanext-dgu#257]
To be really useful, it would be good to allow schema for individual resources as an option to override the dataset one.  This way we can include the schema information in the datapackage.

**Comment by davidread** [comment]
Agreed. I would love to know how you suggest we display this in the dataset form.
